### PR TITLE
Fixed delete copyright from media

### PR DESCRIPTION
**Pull request by wachterjohannes**
__informations:__

| q                | a
| ---------------- | ---
| Fixed tickets    | none 
| Related PRs      | https://github.com/massiveart/husky/pull/587 https://github.com/massiveart/husky/pull/588
| BC breaks        | none
| Documentation PR | none

### CODE REVIEW

File: src/Sulu/Bundle/AdminBundle/Resources/public/js/vendor/husky/husky.js [hunk 1]
```diff
@@ -32323,6 +32322,54 @@ define('husky_components/datagrid/decorators/dropdown-pagination',[],function()
...
+                    if (width > minWidth && width <= maxWidth) {
+                        $element.css({width: width + 'px'});
+                    }
+                }.bind(this)
+            );
+        },
+
+        inputHandler: function(event) {
+            var $element = this.sandbox.dom.find(event.currentTarget),
+                page = parseInt($element.val());
```
Comment (chirimoya) [inline src/Sulu/Bundle/AdminBundle/Resources/public/js/vendor/husky/husky.js]: `page = parseInt($element.val(), 10);`

**Comment by chirimoya** [comment]
update translations

**Comment by chirimoya** [comment]
validate input

**Comment by chirimoya** [comment]
disable button back/forth

**Comment by wachterjohannes** [comment]
i have fixed your comments also in husky (PR https://github.com/massiveart/husky/pull/588). i will create new translation files in master (sulu-standard) after merging this PR

**Pull request merged**